cmd/kops/create.go

```diff
@@ -31,20 +31,37 @@ import (
 	"k8s.io/kops/pkg/apis/kops/v1alpha1"
 	"k8s.io/kops/upup/pkg/fi/cloudup"
 	"k8s.io/kops/util/pkg/vfs"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/util/i18n"
 )
 
 type CreateOptions struct {
 	resource.FilenameOptions
 }
 
+var (
+	create_long = templates.LongDesc(i18n.T(`
+		Create a resource by filename or stdin.`))
+
+	create_example = templates.Examples(i18n.T(`
+		# Create a cluster in AWS
+		kops create cluster --name=k8s.cluster.site \
```

Comment:
Let me take back what I said ;) This is awesome, we need example with a yaml file as well. Ping me an I can get you more info, as I want to be respectful of your time. Take a peek at the PR that was just put in about standards for writing cli docs ;)